contracts/mocks/MockController.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma experimental ABIEncoderV2;
+
+import {Actions} from "../libs/Actions.sol";
+
+contract MockController {
```

Comment:
We shouldn't call this `MockController` since it's just a contract for library testing, not mimicking any operation of the Controller.

Reply:
I suggest calling in `ActionTester`